web/src/utils/helpers.test.ts

```diff
@@ -254,3 +255,34 @@ describe('getNetExchange', () => {
     expect(result).toBe(100);
   });
 });
+
+describe('hasMobileUserAgent', () => {
+  it.each([
+    {
+      userAgent:
+        'Mozilla/5.0 (iPhone; CPU iPhone OS 10_3_1 like Mac OS X) AppleWebKit/603.1.30 (KHTML, like Gecko) Version/10.0 Mobile/14E304 Safari/602.1',
+      expected: true,
+    },
+    {
+      userAgent:
+        'Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Mobile Safari/537.36',
```

Comment:
Should `Mobile` be a part of the regex test?

Reply:
That seems reasonable! 👍